src/runtime/particle-execution-context.ts

```diff
@@ -152,7 +152,7 @@ export class ParticleExecutionContext {
         return new Promise((resolve, reject) =>
           pec.apiPort.ArcMapHandle(id => {
             resolve(id);
-          }, arcId, handle));
+          }, arcId, handle));  // recipe handle vs not?
```

Comment:
Here's a weird case where the passed in Handle does not match the `ArcMapHandle` signatures in `api-channel.ts` and `particle-execution-host.ts`

It appears that the called code only relies on the handle.id, so I guess that's why it works.